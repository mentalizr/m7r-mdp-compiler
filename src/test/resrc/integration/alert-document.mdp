@@name=AutoTest1
@@persistent

### Header 1

Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.

@alert[type="info" headersize="3"]
    Alert Text ...

Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium.