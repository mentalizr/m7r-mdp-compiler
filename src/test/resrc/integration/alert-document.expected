<!--
@@name=AutoTest1
@@persistent
-->

<p class="h3 mt-4 mb-4">Header 1</p>
<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
<div class="alert alert-info mb-4">
    <div style="display: table; width: 100%">
        <div style="display: table-cell; width: 24px; background-image: url(service/v1/mediaImg/info.png); background-repeat: no-repeat; background-position: center center; background-size: contain; width: 24px; height: 24px;"></div>
        <div style="display: table-cell; width: 18px;"></div>
        <div style="display: table-cell;" class="h3">Alert Text ...</div>
    </div>
</div>
<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium.</p>